### DISCUSSION TITLE
I am now somehow a team after migrating to Pro

### DISCUSSION THREAD

**Discussion by NickHodges** [vercel/community#6254] [Account Issue]
### Question

I upgraded to Pro, and now I am somehow a team and I don't want to be a team.

Everything was rolling along great.  I had three projects in my hobby account.  I upgraded to Pro, and now I am somehow a "Team" and I totally do not want that.  I just want everything the same from my Hobby level account to be up at the Pro level.

**Comment by NickHodges** [comment]
Okay, I just deleted my "Team", and I don't think I got charged any money.

This is a really bad user experience.  

I have a great setup here on Vercel.  I like it.  I want to give you all money.  But man, what the hell happens when you want to pay?  Very strange and jarring and unpleasant.

**Comment by amyegan** [comment]
Hey Nick. Thanks for sharing this feedback. I have an idea about what may have happened. But I'd also like to get some more info to make sure I understand what went wrong.

Normally you would see just one new team created when upgrading to pro. Based on your post, it sounds like your hobby account also became a new team at the same time. 

I suspect your account was recently been updated to our new account structure. Essentially, projects on hobby accounts are being moved to free single-person hobby teams with the same abilities and limits. You can read more about the update at https://vercel.com/changelog/2024-01-account-changes.

Please let me know whether I have misinterpreted your post, and share the pro upgrade steps you followed. I want to get the right info to the engineering team so we can fix any bugs in this process. 🙇

**Comment by NickHodges** [comment]
Amy --

I'm afraid I don't have any idea about any type of account.

I just know I had a hobby account.  I wanted a pro account, so I pressed
the "Upgrade to Pro" button, followed the instructions, and I guess I ended
up with a Team account that didn't have anything in it.  It asked to
migrate my existing projects, but when I did that, the projects came over,
but they were blank and dind't have any of the settings (for instance, the
domains were no longer attached.  So I deleted this new team that I had,
and now you have my $20, and I have nothing to show for it.

What I want is for my existing account to remain exactly like it is now,
except I want it to be a Pro account instead of a Hobby account.  I don't
want to form any teams, as it is just me.

Does that make sense?  Can you make that happen?

Just want to point out again that you all have my $20 and I have, well,
nothing new for that $20.  ;-)

Nick

**Comment by NickHodges** [comment]
Amy --

Okay, so here's what I want/expect:

I want my hobby account to have the benefits of the Pro plan, WITHOUT
forming a "team."

But I don't think you provide that.  Am I right?  If you don't, I think you
should. ;-)

Okay, so if I have to form a team, I'd like to be able to form that team
and then add my existing projects to the team as they are.

My experience was that I added my projects to the Team and all the things
associated with the projects were lost, most notably the connection to my
domain name.

In other words, it is unclear to me when I go through the upgrade process
that what I expect is not going to happen and what is going to happen isn't
explained.  You advertise "Upgrade your account to Pro", but what you are
selling is "Create a Team and lose what you had with your hobby account".

Does that make sense?

Nick

**Reply by amyegan** [reply]
Hey Nick. Yes, with the old account structure you would need a new team when you signed up for Pro. With the new structure you have the option to just upgrade your existing projects.

This change was designed to make the upgrade process easier. You're certainly not the only person who didn't like the old Pro upgrade process, and that's why we made the change. I suspect we were just very unlucky between the timing of your Pro upgrade and our migration of your account to the new structure. 

If you click the Upgrade button on the **Settings** > **Billing** tab now, do you see the option to upgrade your existing projects?

<img width="1230" alt="Screenshot" src="https://github.com/vercel/community/assets/8247105/385e1d61-c0dc-488e-a6c0-c8d816952829">

If you haven't already opened a case with our Customer Success team, please [submit an **Invoice Enquiry** case](https://vercel.com/help) so one of our billing admins can refund you for your cancelled upgrade.

**Comment by NickHodges** [comment]
Amy --

Still struggling with this despite my desire to send you money!  :-)

There is no Upgrade button on the *Settings > Billing *tab.

[image: image.png]
Not at all sure how to proceed.

Nick

**Comment by NickHodges** [comment]
Amy --

This just keeps getting harder.

Now, if I go to the Upgrade to Pro screen, it tries to charge me $30, not
$20, adding $10 for something called "Speed Insights" with no way to get
rid of it.

Sigh.

Nick
